src/components/popup/NewPop.jsx

```diff
@@ -27,7 +27,7 @@ function NewPop() {
                 setraised(finalTemp);
                 let val = Number(((raised / 3700000) * 100).toFixed(1));
                 setraisedper(val);
-                console.log(raisedper)
+                console.log(raisedper);
```

Comment:
@roverflow Why is conloe.log added here?

Reply:
We don't need this CL at all in the site, 

![image](https://user-images.githubusercontent.com/23498248/140795681-ea0bbab5-6397-4286-adb1-a1f19eacda93.png)

Reply:
Will be removing all of it in the updated application, working on it locally, will be done by few weeks